resources/schemas/dbscripts/sqlserver/targetedms-19.16-19.17.sql

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2017 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ALTER TABLE targetedms.qcmetricconfiguration ADD PrecursorScoped BIT NOT NULL DEFAULT 1;
```

Comment:
@labkey-ankurj Since you've already done the work to put all of the email notification code in the premium module, what do you think about dropping the subscription table from this module and adding it to the premium side?

Reply:
Done